test/React__test.re

```diff
@@ -589,6 +589,27 @@ describe("React", () => {
     expect(value.contents)->toEqual("My value");
   });
 
+  test("React.Fragment with key", () => {
+    let container = getContainer(container);
+    let title = Some("foo");
+
+    act(() => {
+      ReactDOM.render(
+        <React.Fragment key=title>
```

Comment:
there's definitely a bug in reason-react, but the test is also wrong. this should be `key=?title`

Reply:
`Fragment` and some other special components seem very lax in what they accept. we need to make it only take string (option)s. I'm pushing a fix momentarily.

Reply:
> but the test is also wrong

 Yeah the test looks wrong, but I wanted to reflect the code as it was working before.

Reply:
updated.